core/templates/domain/exploration/LostChangeObjectFactory.ts

```diff
@@ -74,38 +86,46 @@ export class LostChange {
   }
 
   isOutcomeFeedbackEqual() {
-    if (this.newValue.outcome && this.newValue.outcome.feedback &&
-      this.oldValue.outcome && this.oldValue.outcome.feedback) {
+    if ((<ILostChangeValues> this.newValue).outcome &&
+      (<ILostChangeValues> this.newValue).outcome.feedback &&
+      (<ILostChangeValues> this.oldValue).outcome &&
+      (<ILostChangeValues> this.oldValue).outcome.feedback) {
       return (
-        this.newValue.outcome.feedback.getHtml() ===
-        this.oldValue.outcome.feedback.getHtml());
+        (<ILostChangeValues> this.newValue).outcome.feedback.getHtml() ===
+        (<ILostChangeValues> this.oldValue).outcome.feedback.getHtml());
     }
     return false;
   }
 
   isOutcomeDestEqual() {
-    if (this.newValue.outcome && this.oldValue.outcome) {
+    if ((<ILostChangeValues> this.newValue).outcome &&
+      (<ILostChangeValues> this.oldValue).outcome) {
       return (
-        this.oldValue.outcome.dest === this.newValue.outcome.dest);
+        (<ILostChangeValues> this.oldValue).outcome.dest ===
+        (<ILostChangeValues> this.newValue).outcome.dest);
     }
     return false;
   }
 
   isDestEqual() {
-    return this.oldValue.dest === this.newValue.dest;
+    return (<ILostChangeValues> this.oldValue).dest ===
+      (<ILostChangeValues> this.newValue).dest;
   }
 
   isFeedbackEqual() {
-    if (this.newValue.feedback && this.oldValue.feedback) {
+    if ((<ILostChangeValues> this.newValue).feedback &&
+    (<ILostChangeValues> this.oldValue).feedback) {
       return (
-        this.newValue.feedback.getHtml() ===
-        this.oldValue.feedback.getHtml());
+        (<ILostChangeValues> this.newValue).feedback.getHtml() ===
+        (<ILostChangeValues> this.oldValue).feedback.getHtml());
     }
```

Comment:
That is a lot of retyping, can this be somehow simplified to just retype everything at once?

Reply:
I don't think so. We'll have to retype these things in every way I can think.